src/util/initial-config.yaml

```diff
@@ -132,8 +133,10 @@ full_node:
 
   # How often to connect to introducer if we need to learn more peers
   introducer_connect_interval: 500
-  # Continue trying to connect to more peers until this number of connections
-  target_peer_count: 15
+  # Accept peers until this number of connections
+  target_peer_count: 25
```

Comment:
Let's increase this? It was much higher in bitcoin right?

Reply:
Bitcoin uses <= 116 inbound connections and <= 8 outbound connections. Perhaps we can keep the 10 outbound connections, but increase the inbound count to something like 50? (making target_peer_count=60).